ha/const.py

```diff
@@ -69,6 +69,7 @@
 
 # Node name mapping keys
 PVTFQDN_TO_NODEID_KEY="pvtfqdn_to_nodeid"
+NODE_BMC_INFO_KEY="node_bmc_info"
```

Comment:
move this to ipmi fencing agent only

for ha it is abstract detail
all files in utils should be independent of ha

Reply:
Moved to ipmi_fencing_agent class